docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -57,19 +57,20 @@ To enable your application to use the media controller functionality:
 
 ## Updating and Retrieving Information
 
-To update the metadata and playback information on the server side, and to retrieve it on the client side:
+To update the playback information on the server side, and to retrieve it on the client side:
 
--   To update the metadata and playback information on the server side, follow these steps:
+-   To update the playback information on the server side, follow these steps:
     1.  Start the media control server using the `Start()` method of the [Tizen.Multimedia.Remoting.MediaControlServer](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Remoting.MediaControlServer.html) class:
 
         ```csharp
         MediaControlServer.Start();
         ```
 
-    2. Update the metadata or playback information using the `SetMetadata()` or `SetPlaybackState()` methods:
+    2. Update the playback information using the `SetPlaybackState()` method:
 
         ```csharp
         MediaControlServer.SetPlaybackState(MediaControlPlaybackState.Playing, currentPosition);
+        MediaControlServer.SetIndexOfCurrentPlayingMedia("IDX1");
```

Comment:
It's typo. I'll fix it.
`Updating and Retrieving playlist` just used index value. So I think it's not problem.

Reply:
It's typo. I'll change it to SetInfoOfCurrentPlayingMedia.

How about moving `start/stop server and get controller` to Prerequisites?
And move `~~playlist` section before `Updating and retrieving information`.
Because `SetInfoOfCurrentPlayingMedia` requires `playlist` parameter.

Reply:
@mijin85cho , Please review latest PR. I changed the sequence of sections.